pkg/client/factory.go

```diff
@@ -77,18 +78,22 @@ func (f OptimizelyFactory) Client(clientOptions ...OptionFunc) (*OptimizelyClien
 	if f.eventProcessor != nil {
 		appClient.EventProcessor = f.eventProcessor
 	} else {
-		appClient.EventProcessor = event.NewBatchEventProcessor(
+		var eventProcessorOptions = []event.BPOptionConfig{
```

Comment:
do we still need them: 
event.WithBatchSize(event.DefaultBatchSize),
			event.WithQueueSize(event.DefaultEventQueueSize),
			event.WithFlushInterval(event.DefaultEventFlushInterval),
since they use default values ?

Reply:
You are right, we don't need em.